bundles/org.openhab.binding.vigicrues/src/main/java/org/openhab/binding/vigicrues/internal/discovery/VigiCruesDiscoveryService.java

```diff
@@ -37,7 +37,7 @@
  *
  * @author Gaël L'hopital - Initial contribution
  */
-@Component(service = DiscoveryService.class, configurationPid = "discovery.vigicrues")
+@Component(service = DiscoveryService.class)
```

Comment:
You shouldn't have changed this one. You were supposed to remove the one from `ApiHandler` instead.